### ISSUE TITLE
Rebrand: Create t-shirts & stickers

### ISSUE THREAD

**Issue by Mackay031** [kedro-org/kedro-viz#1334]
- [x] Get the icons for the t-shirts and stickers
- [x] Email Eloise to ask for details for a printer
- [ ] Email assets to printer (5XS, 10S, 20M, 15L, 10XL, 5XXL) and 100 stickers

**Comment by yetudada** [comment]
I'm going to mark this as closed. It's been actioned.

**Comment by astrojuanlu** [comment]
We're still blocked on the t-shirts, and I haven't done my part with the stickers. Reopening for tracking purposes.

**Comment by tynandebold** [comment]
We have the t-shirts. Stickers seem secondary? Let's reopen another ticket for those if we still want to track this.
